base/MAPL_AbstractGridFactory.F90

```diff
@@ -82,6 +82,11 @@ module MAPL_AbstractGridFactoryMod
       procedure(get_file_format_vars), deferred :: get_file_format_vars
       procedure(decomps_are_equal), deferred :: decomps_are_equal
       procedure(physical_params_are_equal), deferred :: physical_params_are_equal
+
+      procedure :: get_xy_subset
+      procedure :: get_xy_mask
+      procedure :: destroy
+      procedure :: get_obs_time
```

Comment:
I still hope to find a way eventually to keep anything like "obs time" from leaking into the geometry description.

When the dust settles we should step back and consider how all of this will work in MAPL3.